python/nano/test/automl/tf/test_no_search.py

```diff
@@ -0,0 +1,131 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import sys
+sys.path.append("../../../src/")
+
+import pytest
+from unittest import TestCase
+
+import bigdl.nano.automl as automl
+automl.hpo_config.enable_hpo_tf()
+
+import tensorflow as tf
+import tensorflow.keras as keras
+import numpy as np
+import bigdl.nano.automl.hpo.space as space
+from bigdl.nano.tf.keras import Input
+from bigdl.nano.tf.keras.layers import Dense, Conv2D, Flatten
+from bigdl.nano.automl.tf.keras import Model, Sequential
+
+from tensorflow.keras.optimizers import Adam, RMSprop
+import bigdl.nano.automl.hpo as hpo
+
+@hpo.tfmodel()
+class MyModel(tf.keras.Model):
+    def __init__(self, filters, kernel_size, strides, num_classes=10):
+        super().__init__()
+        self.conv1 = tf.keras.layers.Conv2D(filters=filters,
+                            kernel_size=kernel_size,
+                            strides=strides,
+                            activation="relu")
+        self.max1  = tf.keras.layers.MaxPooling2D(3)
+        self.bn1   = tf.keras.layers.BatchNormalization()
+
+        self.gap   = tf.keras.layers.GlobalAveragePooling2D()
+        self.dense = tf.keras.layers.Dense(num_classes)
+
+    def call(self, inputs, trainig=False):
+        x = self.conv1(inputs)
+        x = self.max1(x)
+        x = self.bn1(x)
+        x = self.gap(x)
+        return self.dense(x)
+
+
+class TestUseCases(TestCase):
+
+    def __init__(self,*args, **kwargs):
+        super(TestUseCases, self).__init__(*args, **kwargs)
+        self.TRAIN_TOTAL_SAMPLES = 3000 # use less data
+        self.VALID_TOTAL_SAMPLES = 1000 # use less data
+        self.TEST_TOTAL_SAMPLES = 1000
+        np.random.seed(1337)
+        self._create_data()
+
+    def _create_data(self):
```

Comment:
Seems you're using the same model and datasets as TestUseCases, Then you can just modify the TestUseCases instead of creating a new test file.